Dockerfile

```diff
@@ -1,49 +1,59 @@
-# syntax=docker/dockerfile:1.3
+# syntax=docker/dockerfile:1
 
-ARG GO_VERSION=1.16.15
-ARG GORELEASER_XX_VERSION=1.2.5
+ARG GO_VERSION=1.18
```

Comment:
Same for this one. If anyone else wants to build from the tag and wants to use a different version of Go, they'll be able to use the build-args to override, so pinning to an exact version should not limit them.

Reply:
updated.